src/app/commissioner_app.cpp

```diff
@@ -1321,6 +1304,39 @@ const JoinerInfo *CommissionerApp::GetJoinerInfo(JoinerType aType, const ByteArr
     return nullptr;
 }
 
+bool CommissionerApp::HandleCommissioning(const JoinerInfo & aJoinerInfo,
+                                          const std::string &aVendorName,
+                                          const std::string &aVendorModel,
+                                          const std::string &aVendorSwVersion,
+                                          const ByteArray &  aVendorStackVersion,
+                                          const std::string &aProvisioningUrl,
+                                          const ByteArray &  aVendorData)
+{
+    (void)aVendorName;
+    (void)aVendorModel;
+    (void)aVendorSwVersion;
+    (void)aVendorStackVersion;
+    (void)aVendorData;
+
+    constexpr auto kJoinerNotAccepted = false;
+    constexpr auto kJoinerAccepted    = true;
+
+    auto configuredJoinerInfo = GetJoinerInfo(aJoinerInfo.mType, Commissioner::ComputeJoinerId(aJoinerInfo.mEui64));
+    if (configuredJoinerInfo == nullptr)
+    {
+        // TODO(wgtdkp): logging
+        return kJoinerNotAccepted;
```

Comment:
@deimi `OpenThread` doesn't encourage multiple `return` in a function, could you consider using `VerifyOrExit(...)`? for example, you can change this to
```c++
VerifyOrExit(configuredJoinerInfo != nullptr, accepted = false);
```

Reply:
I will, but just to mention it, there are several multiple return functions in `commissioner_app.cpp`

Reply:
oh, could you help point out them? I am appreciate if you even like to make a PR to fix it.

Reply:
e.g. `GetJoinerInfo`

Reply:
Thanks, I will fix those multiple returns.